packages/axelar-std/tests/event.rs

```diff
@@ -0,0 +1,199 @@
+use soroban_sdk::testutils::Address as _;
```

Comment:
rebase the PR over main correctly. Here's an easier to rebase since this PR was pointing to your other PR:
- Restore the branch from the merged PR
- Merge that branch onto this PR, which is easier since the commit history is preserved
- Merge latest main onto this PR
- Delete old branch and merge this PR

Reply:
we might want to play around with stacked diffs/graphite